apps/sertra/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Do we want to use app specific lint rules, or should sertra use the ones defined in the root dir? Or extend it?